language.md

```diff
@@ -20,6 +20,8 @@ If you speak Punjabi, [go here](punjabi/punjabi.md).
 
 如果你说中文而且想要了解一个新的世界，[请点击这里](Chinese/DesertHell/NewWorld.md).
 
+如果你说中文而且喜欢玩LOL，[请点击这里](chinese/Goku/Guku_arrives.md)

```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.